docs/core/testing/snippets/unit-testing-using-mstest/csharp/unit-testing-using-mstest.sln

```diff
@@ -0,0 +1,48 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
We don't normally check in solution files?

Reply:
I think we did here because it's a multi-project solution.

Reply:
Yes, that's why. I had to do a force add to get it in.